webrtc.html

```diff
@@ -6623,19 +6623,23 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       keep information from RTP packets received in the previous 10 seconds.
       When the first audio frame contained in an RTP packet is delivered to the
       <code><a>RTCRtpReceiver</a></code>'s <code><a>MediaStreamTrack</a></code>
-      for playout, the relevant <code><a>RTCRtpContributingSource</a></code>
-      and <code><a>RTCRtpSynchronizationSource</a></code> objects are updated. The
-      <code><a>RTCRtpSynchronizationSource</a></code> object corresponding to the
-      SSRC identifier is always updated, and if the RTP packet contains CSRC
-      identifiers, then the <code><a>RTCRtpContributingSource</a></code>
+      for playout, the user agent MUST queue a task to update the relevant
+      <code><a>RTCRtpContributingSource</a></code> and
+      <code><a>RTCRtpSynchronizationSource</a></code> objects based on the
+      contents of the packet. The
+      <code><a>RTCRtpSynchronizationSource</a></code> object corresponding to
+      the SSRC identifier is always updated, and if the RTP packet contains
```

Comment:
I had a bit of trouble parsing "always" here. Maybe s/always updated/updated each time/ to underscore we're talking about frequency and the packet in the previous sentence?